SmartPlaces.Facilities/products/Topology/src/Topology.csproj

```diff
@@ -0,0 +1,54 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Worker">
```

Comment:
based on how we've been talking about this, should this be placed in `.\smartplaces.facilities\samples\topology`?